packages/taboule/src/state/queries.ts

```diff
@@ -93,7 +93,15 @@ export const GetTabouleQueries = ({
         API.v2.Public.GetExperimentList({
           ...input,
         }),
-        TE.map((content) => ({ total: content.length, content }))
+        TE.map((content) => {
+          return {
+            total: content.configured.length,
+            content: content.configured.map((c) => ({
```

Comment:
error, the "configured" field was present in the previous version (which you might have found online)

Reply:
I'm fixing this